gvr-modelviewer2/app/src/main/java/org/gearvrf/modelviewer2/Controller.java

```diff
@@ -37,7 +37,7 @@
     // Variables related to SkyBox
     private ArrayList<SkyBox> aODefaultSkyBox;
     private ArrayList<SkyBox> aOSDSkyBox;
-    private final String sSDSkyBoxDirectory = "GVRModelViewer2/SkyBox";
+    private final String sSDSkyBoxDirectory = "SceneEditor/environments/";
```

Comment:
Is this path intentionally changed?

Reply:
Yes, I changed it so that the model viewer and the scene editor can share the same content on the phone. Eventually I want to merge these applications into a single VR editor. This will take a while...